Nitean recognition.md

```diff
@@ -0,0 +1,27 @@
+# Nitean recognition
+
+As Niteans, we believe in celebrating small wins and recognizing achievements. By enabling Nitean recognition, we're boosting each other's morale, enhancing engagement and communication, and improving the overall company culture.
+
+## Wall of Happy
+
+This is inspired by head of Google’s People Operations, Laszlo Bock. He has a Wall of Happy outside his office where kudos are printed and put on a physical wall.
+
+For our use case, the wall-of-happy is a Slack channel where you can see a steady stream of kudos and reactions, enabled by the [Bonus.ly](https://bonus.ly/) peer recognition platform.
+
+## Policy
+
+1. Niteans get 10 points allowance every month. We're adopting the `+1` system. This means that whenever you give someone a bonus, you give that person 1 point.
+
+```
+/give +1 @nitean for doing a stellar job at designing our homepage
+```
+
+The more +1 (peer recognition) you get, the more points you collect. Instead of reactions, other Niteans who want to acknowledge the recognition can also give their +1 by clicking `Add on this bonus?`.
+
+2. Everyone starts with 0 bonus reward. When you join Bonus.ly, it says you've been awarded 2 points. This is reset to 0.
+
+3. Aside from peer recognition, we're also doing a basic reward system. When Niteans reach a specific amount of points, they can redeem it for a one workday off. We don't believe in monetary incentive, so a one day off and put the feet up is a nice reward for a job well done.
```

Comment:
Bullet points rather than numbers since these are not steps.

Some wording changes:
* `Aside from` should be `In addition to`
* `we're also doing a` should be `we also have a`
* What is the specific amount? 
* Delete `We don't believe in monetary incentive, so a one day off and` it's superfluous and highlights an alternative unavailable benefit.